pages/common/stdbuf.md

```diff
@@ -0,0 +1,15 @@
+# stdbuf
+
+> Run `command`, with modified buffering operations for its standard streams.
```

Comment:
Run a command with modified buffer sizes for its standard streams.

Reply:
:heavy_check_mark:

Reply:
I'm fine with the change. However, the original description was taken directly from the stdbuf(1) man page in coreutils:

```
STDBUF(1)                        User Commands                       STDBUF(1)

NAME
       stdbuf  - Run COMMAND, with modified buffering operations for its stan‐
       dard streams.
```